app-backend/db/src/main/resources/migrations/V4__Task_and_Task_Actions.sql

```diff
@@ -0,0 +1,33 @@
+-- Create tasks table
+
+CREATE TYPE task_status AS ENUM (
+    'UNLABELED', 'LABELING_IN_PROGRESS', 'LABELED', 'VALIDATION_IN_PROGRESS', 'VALIDATED'
+);
+
+CREATE TABLE tasks (
+  id uuid primary key,
+  created_at timestamp without time zone  not null,
+  created_by text not null references users (id),
+  modified_at timestamp without time zone not null,
+  modified_by text not null,
+  project_id uuid not null,
+  project_layer_id uuid not null,
+  status task_status not null,
+  locked_by text references users (id),
+  locked_on timestamp without time zone,
+  geometry geometry(Geometry, 3857),
+  CONSTRAINT tasks_project_id_fkey FOREIGN KEY (project_id) REFERENCES projects (id)
```

Comment:
one consequence of the decision that tasks with actions can't be deleted is that now projects (and project layers) with tasks with actions can't be deleted. I don't know if that matters